src/VisualStudio/Setup/AssemblyRedirects.cs

```diff
@@ -3,6 +3,14 @@
 using Microsoft.VisualStudio.Shell;
 
 [assembly: ProvideBindingRedirection(
+    AssemblyName = "Microsoft.Build.Tasks.Roslyn",
```

Comment:
It should be Microsoft.Build.Tasks.CodeAnalysis.  That is our shipping DLL naming pattern now.